dlp/nox.py

```diff
@@ -18,35 +18,9 @@
 import nox
 
 
-LOCAL_DEPS = (
-    os.path.join('..', 'api_core'),
-    os.path.join('..', 'core'),
-)
-
-
 @nox.session
 def default(session):
-    """Run the unit test suite.
-
-    This is intended to be run **without** an interpreter set, so
-    that the current ``python`` (on the ``PATH``) or the version of
-    Python corresponding to the ``nox`` binary the ``PATH`` can
-    run the tests.
-    """
-    session.install('mock', 'pytest', 'pytest-cov', *LOCAL_DEPS)
-    session.install('-e', '.')
-
-    session.run(
-        'py.test',
-        '--quiet',
-        '--cov=google.cloud.dlp_v2',
-        '--cov-append',
-        '--cov-config=.coveragerc',
-        '--cov-report=',
-        '--cov-fail-under=97',
-        os.path.join('tests', 'unit'),
-        *session.posargs
-    )
+    return unit(session, 'default')
```

Comment:
I don't understand why we are dropping the `system` / `lint` `cover` bits, nor why this library would be different than any of our other ones.

Reply:
this is the template that comes from our generator. I can revert this part since it seems to be dropping things.